build.gradle

```diff
@@ -233,13 +238,13 @@ compileJava9Java {
 }
 
 shadowJar {
-	minimize() {
+	/*minimize() {
```

Comment:
why not minimize anymore?

Reply:
Oops, sorry, that was a test that shouldn't have been committed...